opennms-doc/guide-admin/src/asciidoc/text/bsm/bsm-admin.adoc

```diff
@@ -0,0 +1,125 @@
+|===
+
+==== Calculation of the operational status
+
+The operational status of a _Business Service_ is correlated in two steps.
+First, the status of each contained element is mapped to the _Business Service_.
+For _Reduction Keys_ and _IP Services_ the status is based on their alarm status.
+For contained _Business Services_ the operational status is used.
+
+===== Map Functions
+
+The mapping can be performed by one of _Map Functions_ listed below:
+
+.State calculation _map functions_
+[options="header, autowidth"]
+|===
+| Name       | Description
+| `Identity` | Use the status according to the reported severity
+| `Increase` | Use the status according to the reported severity increased by one level
+| `Decrease` | Use the status according to the reported severity decreased by one level
+| `SetTo`    | Uses a constant status value
+| `Ignore`   | Ignores the severity of the managed entity
+|===
+
+===== Reduce Funtions
+
+Finally, the list of mapped status values are then aggregated into the operational status of the _Business Service_.
+For this operation the following _Reduce Functions_ are available:
+
+.Status calculation _Reduce Functions_
+[options="header, autowidth"]
+|===
+| Name            | Description
+| `Most Critical` | Uses the value of the most critical status.
+| `Threshold`     | Uses the highest severity found more often than the given threshold.
+|===
+
+
+===== Operational status changes
+If a _Business Service_ changes its operational status an _OpenNMS_ event of the type `uei.opennms.org/bsmd/serviceOperationalStatusChanged` is fired.
+The log message of the event contains the name of the _Business Service_.
+Additionally, the previous operational status and the new operational status are included.
+Please note, that this event has no node, interface or service associated.
+
+===== The Business Service Daemon
+Internally the _Business Service Monitoring_ feature is driven by the _Business Service Daemon_ (bsmd).
+It is responsible for tracking the operational status of all _Business Services_ and for sending events in case of operational status changes.
+Every time the configuration of a _Business Service_ is changed a reload of the daemon's configuration is required.
+This includes changes like the name of the _Business Service_ or its attributes as well as changes regarding the _Reduction Keys_, contained _Business Services_ or _IP Services_.
+
+====== Reloading Business Service Daemon configuration
+A configuration reload can be initiated by an `uei.opennms.org/internal/reloadDaemonConfig` event with the parameter `daemonName` set to `bsmd`.
+If the reload of the configuration is done an event of type `uei.opennms.org/internal/reloadDaemonConfigSuccessful` is fired.
+The following options to trigger a configuration reload are available:
+
+- Use the `Reload` button on the `Manage Business Services` page in the administration section of the web UI.
```

Comment:
Sometimes a bullet point is a sentence with "." sometimes not and most have just one bullet point.